awx/main/credential_plugins/hashivault.py

```diff
@@ -85,8 +89,21 @@
     'type': 'string',
     'help_text': _('Valid principals (either usernames or hostnames) that the certificate should be signed for.'),
 }]
-hashi_ssh_inputs['required'].extend(['public_key', 'role'])
+hashi_ssh_inputs['required'].extend(['public_key', 'secret_path', 'role'])
```

Comment:
Just to make sure I've got this right - the *new* Hashi Token *doesn't* have a `secret_path` at all, so that's why you've pulled it out and put it into the KV and SSH plugins explicitly?

Reply:
Yes, token is the default engine and it's not possible to set a path, it's always /token